test/unit-tests/inputField.test.js

```diff
@@ -0,0 +1,115 @@
+let { openBrowser, goto, inputField, closeBrowser, intervalSecs, timeoutSecs, below } = require('../../lib/taiko');
+let { createHtml, openBrowserArgs, removeFile } = require('./test-util');
+const test_name = 'inputfield';
+
+describe('inputField', () => {
+
+    beforeAll(async () => {
+        await openBrowser(openBrowserArgs);
+    }, 30000);
+
+    afterAll(async () => {
+        await closeBrowser();
+    }, 30000);
+
+    describe('with inline text', () => {
```

Comment:
`describe` can be `attribute and value pair` since there is no `inline text` here